### DISCUSSION TITLE
support OpenAPI v3

### DISCUSSION THREAD

**Discussion by lucklyt** [swaggo/swag#1749] [General]
is it  possible  to support OpenAPI v3?

**Comment by losinggeneration** [comment]
#548 includes some details on the effort for v3 support. TL;DR - Check the v2 branch or the v2.x tags for the work in progress support.

**Reply by MrVermond** [reply]
how to install such a v2 tag?

**Comment by 8thgencore** [comment]
The repository is dying, as for me, it looks promising https://github.com/danielgtaylor/huma
